src/components/OurStory.tsx

```diff
@@ -0,0 +1,21 @@
+
+
+const OurStory = () => {
+
+    const headText = "Our Story"
+    const infoText = "HARBS is a café born in Japan, pursuing freshness and hand-made quality above all else. Ever since we started, our one and only wish has been to make our customers say “DELICIOUS!” from the bottom of their hearts. We hope you savor every moment of joy you experience here at HARBS."
+
+    return (
+    <div className="flex flex-wrap items-start flex-col bg-harbs-yellow text-center">
+        <div className="mx-10 my-10">
+            <h2 className="text-6xl font-bold text-harbs-brown font-cormorant-garamond">{headText}</h2>
```

Comment:
this font size is a little big, because i want the hero text to be larger than this one's.
maybe 5xl for desktop, and then also need to make it smaller on mobile, so maybe mobile is 3 or 4? not sure.